src/Raven.Server/Documents/Replication/ConflictManager.cs

```diff
@@ -137,14 +146,26 @@ public static void AssertChangeVectorNotNull(string conflictedChangeVector)
                 InvalidConflictWhenThereIsNone(conflict.Id);
 
             conflictedDocs.Add(conflict.Clone());
+            conflictedDocs = conflictedDocs.OrderBy(x => x.Doc?.GetHashCode()).ToList();
```

Comment:
This is effectively random, why do you do this?
Also, it isn't guaranteed to be the same on different nodes

Reply:
Right... we need some deterministic way to order them.
Maybe rely on the CV for that? (But I guess we need to order it internally as well)

Reply:
Ordering by CV will be a good idea, yes.